src/components/WebexMember/WebexMember.stories.js

```diff
@@ -34,6 +34,13 @@ Host.args = {
   personID: 'user4',
 };
 
+export const Guest = Template.bind({});
+Guest.args = {
+  destinationType: 'meeting',
+  destinationID: 'meeting2',
+  personID: 'user5',
```

Comment:
Maybe use `user6` from this same meeting? [You've already update the data like that anyway](https://github.com/webex/components/pull/197/files#diff-202791f48524a3e98537bfa5bb91dbf5e6e849322b0b5abda6faafeea9f8e089R170).
The only concern I have with using `user5` for this, is that `user5` is also the external user, and I don't want developers confusing "guest" vs "external"